templates/environment/versions/cf-v1.5.0.yml

```diff
@@ -0,0 +1,387 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+Description: CloudFormation environment template for infrastructure shared among Copilot workloads.
+Metadata:
+  Version: 'v1.5.0'
+Parameters:
+  AppName:
+    Type: String
+  EnvironmentName:
+    Type: String
+  ALBWorkloads:
+    Type: String
+    Default: ""
+  EFSWorkloads:
+    Type: String
+    Default: ""
+  NATWorkloads:
+    Type: String
+    Default: ""
+  ToolsAccountPrincipalARN:
+    Type: String
+  AppDNSName:
+    Type: String
+    Default: ""
+  AppDNSDelegationRole:
+    Type: String
+    Default: ""
+  Aliases:
+    Type: String
+    Default: ""
+  ServiceDiscoveryEndpoint:
+    Type: String
+    Default: ""
+Conditions:
+  CreateALB:
+    !Not [!Equals [ !Ref ALBWorkloads, "" ]]
+  DelegateDNS:
+    !Not [!Equals [ !Ref AppDNSName, "" ]]
+  ExportHTTPSListener: !And
+    - !Condition DelegateDNS
+    - !Condition CreateALB
+  CreateEFS:
+    !Not [!Equals [ !Ref EFSWorkloads, ""]]
+  CreateNATGateways:
+    !Not [!Equals [ !Ref NATWorkloads, ""]]
+  HasAliases:
+    !Not [!Equals [ !Ref Aliases, "" ]]
+  LegacyServiceDiscovery:
+    !Equals [ !Ref ServiceDiscoveryEndpoint, "" ]
+Resources:
+{{- if not .ImportVPC}}
+{{include "vpc-resources" .VPCConfig | indent 2}}
+{{include "nat-gateways" .VPCConfig | indent 2}}
+{{- end}}
+  # Creates a service discovery namespace with the form provided in the parameter.
+  # For new environments after 1.5.0, this is "env.app.local". For upgraded environments from 
+  # before 1.5.0, this is app.local.
+  ServiceDiscoveryNamespace:
+    Type: AWS::ServiceDiscovery::PrivateDnsNamespace
+    Properties:
+        Name: 
+          !If
+            - LegacyServiceDiscovery
+            - !Sub ${AppName}.local
```

Comment:
can we possibly remove the conditional and pass in `appname.local` as a parameter value from the cli?

Reply:
done!